src/components/ui/GrommetUI.jsx

```diff
@@ -0,0 +1,544 @@
+                            </Avatar>
+                            <Box>
+                                <Heading margin='0' level='4'>
+                                    {item}
+                                </Heading>
+                                <Heading margin='0' color='status-unknown' level='6'>
+                                    Not reminded
+                                </Heading>
+                            </Box>
+                        </Box>
+                    </Card> 
+                ))}
+            </Box>
+        </Box>
+)};
+
+const EditAction = props => {
+    const defaultOptions = [];
+    for (let x = 1; x <= 10; x += 1) {
+        defaultOptions.push(`Campaign ${x}`);
+    }
+    const [options, setOptions] = useState(defaultOptions);
+    const [value, setValue] = React.useState('');
+    const [valueMultiple, setValueMultiple] = useState([]);
+    return (
+        <Box fill='horizontal' margin='medium'>
+            <Heading level='3'>
+                Edit action
+            </Heading>
+            <Heading level='6'>
+                Campaign
+            </Heading>
+            {/* It's possible to limit selection */}
+            <Select
+                multiple
+                size='medium'
+                placeholder='Start typing to find or create'
+                value={valueMultiple}
+                options={options}
+                onChange={({ value: nextValue }) => setValueMultiple(nextValue)}
+                onClose={() => setOptions(defaultOptions)}
+                onSearch={text => {
+                    // The line below escapes regular expression special characters:
+                    // [ \ ^ $ . | ? * + ( )
+                    const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
+                    // Create the regular expression with modified value which
+                    // handles escaping special characters. Without escaping special
+                    // characters, errors will appear in the console
+                    const exp = new RegExp(escapedText, 'i');
+                    setOptions(defaultOptions.filter(o => exp.test(o)));
+                }}
+            />
+            <Heading level='6'>
+                Date
+            </Heading>
+            <DateInput
+                margin={{'horizontal': 'medium'}}
+                format='mm/dd/yyyy'
+                value={(new Date()).toISOString()}
+                onChange={({ value }) => {}}
+            />
+            <Box direction='row'>
+                <Box>
+                    <Heading level='6'>
+                        Start time
+                    </Heading>
+                    <MaskedInput
+                        mask={[
+                            {
+                                length: [1, 2],
+                                options: Array.from({ length: 24 }, (v, k) => k + 1),
+                                regexp: /^1[0,1-2]$|^0?[1-9]$|^0$/,
```

Comment:
This regexp and time range works poorly with a 24-hour clock. The first regexp should allow 0-23, e.g.  `/^(0|1)?[0-9]$|^2[0-3]$/`